assets/js/base/components/product-list/container.js

```diff
@@ -54,6 +52,4 @@ ProductListContainer.defaultProps = {
 	product_page: 1,
```

Comment:
We don't need to have `defaultProps` for functional components, right? Because the default prop values are already defined in the function signature.